oracle/Assets/Scripts/DelphiClient.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+using SimpleJSON;
+
+public class DelphiClient : MonoBehaviour
+{
+  //   todo: remove
+  //   private readonly string EX_GET_REQUEST_URL = "https://dog.ceo/api/breed/akita/images/random";
+  //   private readonly string EX_POST_REQUEST_URL = "https://reqres.in/api/users";
+
+  // todo: update w/ actual url
+  private readonly string GET_NODES_URL = "";
```

Comment:
Is there a way we can pass command line arguments to the Unity app to give it the url of delphi? @mooncollin

Reply:
using a config file